io.github.prateekmedia.appimagepool.yml

```diff
@@ -0,0 +1,56 @@
+app-id: io.github.prateekmedia.appimagepool
+runtime: org.gnome.Platform
+runtime-version: "40"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+rename-icon: appimagepool
+rename-appdata-file: appimagepool.appdata.xml
+rename-desktop-file: appimagepool.desktop
+command: appimagepool
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --persist=Applications
+  - --talk-name=org.freedesktop.Notifications
+build-options:
+  append-path: /usr/lib/sdk/rust-stable/bin:/app/lib/sdk/flutter/bin
+  env:
+    CARGO_HOME: /run/build/appimagepool/cargo
+modules:
+  - name: flutter
```

Comment:
I don't know anything about Flutter, but if the resulting binary is fully static, you should add `cleanup: ["*"]`.